internal/xds/translator/testdata/out/xds-ir/jwt-ratelimit.clusters.yaml

```diff
@@ -40,6 +40,28 @@
   outlierDetection: {}
   perConnectionBufferLimitBytes: 32768
   type: EDS
+- commonLbConfig:
```

Comment:
why did this config change ?

Reply:
The same configuration, just the `commonLbConfig` location changed in its parent config. It's because of this line: 

```
		// Add all the other needed resources referenced by this filter to the
		// resource version table.
		if err := patchResources(tCtx, httpListener.Routes); err != nil {
			return err
		}

		// RateLimit filter is handled separately because it relies on the global
		// rate limit server configuration.
		// Check if a ratelimit cluster exists, if not, add it, if it's needed.
		if err := t.createRateLimitServiceCluster(tCtx, httpListener); err != nil {
			errs = multierror.Append(errs, err)
		}
```

Reply:
The same configuration, just the `commonLbConfig` location changed in its parent config. It's because of this line: 

```
		// Add all the other needed resources referenced by this filter to the
		// resource version table.
		if err := patchResources(tCtx, httpListener.Routes); err != nil {
			return err
		}

		// RateLimit filter is handled separately because it relies on the global
		// rate limit server configuration.
		// Check if a ratelimit cluster exists, if not, add it, if it's needed.
		if err := t.createRateLimitServiceCluster(tCtx, httpListener); err != nil {
			errs = multierror.Append(errs, err)
		}
```